lib/utils.js

```diff
@@ -109,28 +117,40 @@ function requestWithRetry(request, options, callback) {
  * the details of working with auth in the client code.
  *
  * @param {Array<string>} scopes list of scopes to request as part of auth
+ * @param {Object} config an object with extra configuration parameters (such
+ *     as keyFile or key)
  * @return {function(Object, function(=?,=?,=?):?)} request style function
  *     accepting (options, callback)
  */
-function authorizedRequestFactory(scopes) {
-
+function authorizedRequestFactory(scopes, config) {
+  // The AuthClient instance associated with each request
```

Comment:
The term `request` may not be the best one to use in this comment. Perhaps use the term `instantiation`.